packages/firestore/src/local/indexeddb_persistence.ts

```diff
@@ -261,7 +228,7 @@ export class IndexedDbPersistence implements Persistence {
       );
     }
 
-    this.referenceDelegate = new IndexedDbLruDelegate(this, lruParams);
+    this.referenceDelegate = new IndexedDbLruDelegateImpl(this, lruParams);
```

Comment:
Should there be a `newIndexedDbLruDelegate` to hide this and mirror other splits like this?